index.js

```diff
@@ -9,6 +9,10 @@ var mkdirp = require('mkdirp');
 var md5 = require('md5');
 var stripAnsi = require('strip-ansi');
 
+// Save timer references so that times are correct even if Date is stubbed.
+// See https://github.com/mochajs/mocha/issues/237
+var Date = global.Date;
```

Comment:
I noticed that mocha uses this in its own reporters, so figured we should do it too.